File: prime_gen.py
def prime_gen(max):
	primes = [2,3,5,7,11,13]
	candidate = 13
	while (True):
		length = len(primes)
		candidate += 2
		index = 0
		if (length == max):
			return primes
		while(True):
			if index == length:
				primes.append(candidate)
				break
			if (candidate % primes[index]) == 0:
				break
			index += 1

print(prime_gen(10000))

File: scripts/prime_gen.rs
fn prime_gen(max: usize) -> Vec<usize> {
	let mut primes = vec![2,3,5,7,11,13];
	let mut candidate = 13;
	loop {
		let length = primes.len();
		candidate += 2;
		let mut index = 0;
		if length == max {
			return primes;
		}
		loop {
			if index == length {
				primes.push(candidate);
				break;
			}
			if (candidate % primes[index]) == 0 {
				break;
			}
			index += 1;
		}
	}
}

fn main() {
	println!("{:?}", prime_gen(112500));
}